packages/twenty-server/src/engine/core-modules/environment/converters/config-var-value.converter.ts

```diff
@@ -0,0 +1,83 @@
+import { EnvironmentVariables } from 'src/engine/core-modules/environment/environment-variables';
+import { TypedReflect } from 'src/utils/typed-reflect';
+
+/**
+ * Handles type conversions for environment variables
+ * when storing and retrieving them from the database.
+ */
+export class ConfigVarValueConverter {
+  /**
+   * Convert a database value to the appropriate type for the environment variable
+   */
+  static toAppType<T extends keyof EnvironmentVariables>(
```

Comment:
I haven't seen static functions in the code base before, doesn't seem like a pattern we use much. We either use utils or service with dependency injection